array/view.mbt

```diff
@@ -27,3 +27,57 @@ pub fn each[T](self : ArrayView[T], f : (T) -> Unit) -> Unit {
     f(self[i])
   }
 }
+
+///|
+pub fn eachi[T](self : ArrayView[T], f : (Int, T) -> Unit) -> Unit {
+  for i, v in self {
+    f(i, v)
+  }
+}
+
+///|
```

Comment:
It would be nice to have an example for this (`all`) and for `eachi` above as well.